src/main/java/baseball/Validation.java

```diff
@@ -0,0 +1,103 @@
+package baseball;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Validation {
+
+    public boolean runException(String num, String option) {
+        boolean flag = true;
+            if (!isValidate(num, option)) {
+                flag = false;
+                throw new IllegalArgumentException("유효하지 않은 값을 입력하셨습니다. 게임을 종료합니다.");
+            }
+        return flag;
+    }
+
+    public boolean isValidate(String num, String option) {
+        boolean flag = true;
+        // 게임 진행중 입력
+        if (option.equals("gaming")) {
+            if (!isNotNull(num) || !isDigitInput(num) || !isThreeLength(num) || !isEachUnique(num)) {
+                flag = false;
+            }
+        }
+        // 게임 종료 후 입력
+        else if (option.equals("game over")) {
+            if (!isNotNull(num) || !isDigitInput(num) || !isOneLength(num) || !isOneOrTwo(num)) {
+                flag = false;
+            }
+        }
+        return flag;
+    }
+
+    public boolean isNotNull(String num) {
+        boolean flag = true;
+
+        if (num == null || num.isBlank()) {
+            flag = false;
+        }
+
+        return flag;
+    }
```

Comment:
아래처럼 한 줄로 축약할 수 있을 것 같아요! 다른 메서드에도 `flag` 변수를 사용하셨는데, `flag` 변수를 꼭 사용하지 않아도 될 것 같아요.

```suggestion
    public boolean isNotNull(String num) {
        return num != null && !num.isBlank();
        // return num != null && num.isNotBlank();
    }
```

Reply:
그렇군요 단순히 true || false 만 반환하면 되니.. 많이 배워갑니다!